bulldozer/evaluate.go

```diff
@@ -37,7 +37,18 @@ func IsPRIgnored(ctx context.Context, pullCtx pull.Context, config Signals) (boo
 // IsPRTriggered returns true if the PR is identified as triggered,
 // false otherwise. Additionally, a description of the reason will be returned.
 func IsPRTriggered(ctx context.Context, pullCtx pull.Context, config Signals) (bool, string, error) {
-	matches, reason, err := config.Matches(ctx, pullCtx, "triggered")
+	matches, reason, err := config.MatchesAny(ctx, pullCtx, "triggered")
+	if err != nil {
+		// trigger must always fail closed (no match on error)
+		return false, reason, err
```

Comment:
Not really part of your change, but I just noticed that `IsPRIgnored` doesn't match the implementation of these other two `Is__` methods - might be nice to have them do the same thing (either update `matches` or return directly)